src/shims/unix/freebsd/foreign_items.rs

```diff
@@ -63,6 +64,33 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriInterpCxExt<'mir, 'tcx> {
                 this.write_scalar(Scalar::from_target_usize(len, this), dest)?;
             }
 
+            // File related shims
+            // For those, we both intercept `call` and `call@FBSD_1.0` symbols cases
+            // since freebsd 12 they can be expected
```

Comment:
```suggestion
            // For those, we both intercept `func` and `func@FBSD_1.0` symbols cases
            // since freebsd 12 they can be expected
```

I don't understand the second line. Who can expect what since freebsd 12?